pkgs/jni/src/internal.h

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+#pragma once
+
+#include "dartjni.h"
```

Comment:
Maybe some documentation of why we have internal.h and dartjni.h?

Is it that users don't have to copy in the internal.h ?